src/workflows/airqo_etl_utils/airqo_utils.py

```
@@ -1036,33 +1036,53 @@ def calibrate_data(data: pd.DataFrame) -> pd.DataFrame:
+                logger.exception(f"Error while consuming message: {e}")
+            continue
+
+        try:
+            devices = pd.DataFrame(devices_list)
+            # Will be removed in the future. Just here for initial tests.
+            devices.drop(
+                devices.columns[devices.columns.str.contains("^Unnamed")],
+                axis=1,
+                inplace=True,
+            )
+        except Exception as e:
```

Ruff

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)